packages/stylist/Cargo.toml

```diff
@@ -48,16 +48,12 @@ trybuild = "1.0.53"
 yew = "0.19.3"
 
 [features]
+default = ["macros", "yew_integration", "random"]
 random = ["fastrand", "instant"]
 macros = ["stylist-macros"]
 parser = ["stylist-core/parser"]
-default = ["macros", "parser", "random"]
```

Comment:
"parser" was enabled by default as it provides check after the interpolated values are inserted into the stylesheet when debug builds are enabled.

I personally think this is worth it to be a default weight unless we have a good way to check the interpolated values separately.

What do you think?

Reply:
There definitely *that* part has some worth. What I want to avoid is a user accidentally using runtime parsing, for example by using a string argument to `<Global css`. There are a few other instance that get enabled with `parser` that I think are problematic. I'll try to come up with a solution that also enables the debug_assertions.